bundles/org.openhab.binding.qbus/src/main/java/org/openhab/binding/qbus/internal/QbusBridgeHandler.java

```diff
@@ -0,0 +1,360 @@
+        if (qbusCommunication != null) {
+            qbusCommunication.setBridgeCallBack(this);
+        }
+    }
+
+    /**
+     * Create communication object to Qbus server and start communication.
+     *
+     * @param addr : IP address of Qbus server
+     * @param port : Communication port of QbusServer
+     */
+    private void createCommunicationObject(InetAddress addr, int port) {
+        scheduler.submit(() -> {
+
+            setQbusCommunication(new QbusCommunication(thing));
+
+            QbusCommunication qbusCommunication = getQbusCommunication();
+
+            setBridgeCallBack();
+            if (qbusCommunication != null) {
+                try {
+                    qbusCommunication.startCommunication();
+                } catch (InterruptedException | IOException e) {
+                    bridgeOffline("Communication could not be established {}" + e.getMessage());
+                    return;
+                }
+
+                if (!qbusCommunication.communicationActive()) {
+                    bridgeOffline("No communication with Qbus Server");
+                    return;
+                }
+
+                if (!qbusCommunication.clientConnected()) {
+                    bridgeOffline("No communication with Qbus Client");
+                    return;
+                }
+            }
+
+            updateStatus(ThingStatus.ONLINE);
+
+        });
+    }
+
+    /**
+     * Take bridge offline when error in communication with Qbus server. This method can also be
+     * called directly from {@link QbusCommunication} object.
+     */
+    public void bridgeOffline(String message) {
+        updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.COMMUNICATION_ERROR, message);
+    }
+
+    /**
+     * Put bridge online when error in communication resolved.
+     */
+    public void bridgeOnline() {
+        updateStatus(ThingStatus.ONLINE);
+    }
+
+    /**
+     * Schedule future communication refresh.
+     *
+     * @param interval_config Time before refresh in minutes.
+     */
+
+    private void setupRefreshTimer(int refreshInterval) {
+        ScheduledFuture<?> timer = refreshTimer;
+        if (timer != null) {
+            timer.cancel(true);
+            refreshTimer = null;
+        }
+
+        if (refreshInterval == 0) {
+            return;
+        }
+
+        // This timer will check connection with server and client periodically
+        refreshTimer = scheduler.scheduleWithFixedDelay(() -> {
+            QbusCommunication comm = getCommunication();
+
+            if (comm != null) {
+                if (!comm.communicationActive()) {
+                    // Disconnected from Qbus Server, try to reconnect
+                    try {
+                        comm.restartCommunication();
+                    } catch (InterruptedException e) {
+                        bridgeOffline("No connection with Qbus Server" + e.toString());
```

Comment:
There needs to be a space. Please check all.
```suggestion
                        bridgeOffline("No connection with Qbus Server: " + e.toString());
```